docs/get-started/backup/on_demand_backups.md

```diff
@@ -0,0 +1,158 @@
+# Working with On-demand backups
+
+## [Make a backup](#make-a-backup)
+
+To create a backup:
+
+1. Go to <i class="uil uil-history"></i> **Backup > All Backups**.
+2. Click <i class="uil uil-plus-square"></i> **Create Backup**.
+3. Specify the type of backup that you want to create: **On Demand** or **Schedule Backup**.
+4. Enter a unique name for this backup.
+5. Choose the service to back up from the Service name drop-down menu. This automatically populates the **DB Technology** field.
+6. Select whether you want to create a **Physical** or **Logical** backup of your data, depending on your use case and requirements.
+7. Choose a storage location for the backup. MySQL currently only supports storing backups to Amazon S3. If no options are available here, see the [Create a storage location](prepare_storage_location.md) section above.
```

Comment:
First, we're describing mongo backups and shouldn't mention mysql. Second, we should use S3-compatible instead of Amazon S3.